include/g_compensation/g_compensator_class.h

```diff
@@ -60,6 +60,14 @@ class GCompensator
             buffer_size_ = 50; //default value
             ROS_WARN("Continuing with default buffer_size '50'.");
         };
+
+        bool negate_wrench_flag_;
```

Comment:
The trailing `_` should only apply for class member variables. So, this variable should be renamed to

`negate_wrench_flag`

Reply:
I did the renaming in [cb6439a](https://github.com/ipa320/g_compensator/pull/4/commits/cb6439ac7c73a3465ba66bab6997a7009ee3cb79).